platform/os_linux.go

```
@@ -94,3 +94,8 @@ func SetOutboundSNAT(subnet string) error {
 func ClearNetworkConfiguration() (bool, error) {
 	return false, nil
 }
+
+func KillProcessByName(processName string) {
```

don't we want to return the error from ExecuteCommand?